app/assets/stylesheets/welcome.scss

```diff
@@ -1,29 +1,34 @@
-// .vidcontainer {
-//   position: absolute;
-//   width: 100%;
-//   top: 50%;
-//   left: 50%;
-//   margin: -100px 0 0 0;
-// }
+.vidcontainer {
+  position: absolute;
+  width: 100%;
+  top: 50%;
+  left: 50%;
+  margin: 0 0 0 0;
+}
 
 video {
-  position: relative;
+  position: fixed;
   left: 50%;
   width: 100%;
+  margin-top: 39px;
    height: auto;
    z-index: -100;
    -ms-transform: translateX(-50%) translateY(-50%);
    -moz-transform: translateX(-50%) translateY(-50%);
    -webkit-transform: translateX(-50%) translateY(-50%);
    transform: translateX(-50%) translateY(-50%);
-   background: url(polina.jpg) no-repeat;
    background-size: cover;
 }
 
+.welcome-text {
+  margin: 400px 0 0 0;
+}
+
 #welcome-title {
+  text-shadow: 2px 2px #888888;
```

Comment:
Color literals like `#888888` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered font-size, margin, text-align, text-shadow